rpcd/playbooks/roles/rpc_maas/templates/elasticsearch_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
```

Comment:
I'm not super familiar with MaaS YAML conventions, but the indentation looks weird here. I see both right and left justifications, with extraneous spaces throughout. Probably should be cleaned up a bit.